src/theory/quantifiers/ematching/ho_trigger.cpp

```diff
@@ -112,7 +112,7 @@ void HigherOrderTrigger::collectHoVarApplyTerms(
           bool curWithinApply = withinApply[cur];
           visited[cur] = Node::null();
           visit.push_back(cur);
-          for (unsigned j = 0, size = cur.getNumChildren(); j < size; j++)
+          for (unsigned j = 0, sizec = cur.getNumChildren(); j < sizec; j++)
```

Comment:
Can you convert this into a for each loop?

Reply:
I can't convert this one since the loop body depends on the value of j.